task60_PageFactory/src/test/java/tut/by/tests/AuthorizationTest.java

```diff
@@ -0,0 +1,71 @@
+package tut.by.tests;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import tut.by.pages.HomePage;
+import tut.by.pages.LoginPage;
+
+import java.io.File;
+
+import static org.testng.Assert.assertEquals;
+
+public class AuthorizationTest {
+    static String url = "https://www.tut.by/";
+    private static final String username = "seleniumtests@tut.by";
+    private static final String password = "123456789zxcvbn";
+    private static final String usertext = "Selenium Test";
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void setUp(){
+        driver = new ChromeDriver();
+        driver.get(url);
+        driver.manage().window().maximize();
+    }
+
+    @AfterMethod
+    public void endUp(){
+        driver.close();
+    }
+
+    @Test
+    public void loginTest() throws Exception {
+        HomePage homePage = new HomePage(driver);
+        homePage.openLoginWindow();
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.logIn(username,password);
+
+        this.takeScreenshot(driver,"E://homePageLogin.png");
```

Comment:
It will work only on windows with existing disk E. Please update with relatives paths to target/ folder

Reply:
done